release.sh

```diff
@@ -128,7 +129,11 @@ revert_tag() {
 
 handle_package_manager () {
 	# script or instructions to push to package manager
-	echo -e "\\nYou should now publish this version to the appropriate package manager"
+	release_title="[RELEASE] ${new_version}"	# XXX: this should be urlencoded
```

Comment:
Any reason not to just implement a `urlencode`?

Reply:
I think it's because `urlencode` is not guaranteed to be present on the release-creating hosts.